account_invoice_report_stock_lot/models/res_config_settings.py

```diff
@@ -0,0 +1,10 @@
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    group_lot_on_invoice = fields.Boolean(
+        "Display Lots & Serial Numbers on Invoices",
+        implied_group='account_invoice_report_stock_lot.group_lot_on_invoice'
+    )
```

Comment:
```suggestion
    group_lot_on_invoice = fields.Boolean(
        "Display Lots & Serial Numbers on Invoices",
        help="Lots & Serial numbers will appear on the invoice",
        implied_group="account_invoice_report_stock_lot.group_lot_on_invoice",
    )
```

Reply:
Okay @ivantodorovich 

noted it.

Reply:
Hi @ivantodorovich 

I just pushed your suggested changes.